.github/workflows/auto-assign-to-project.yml

```
@@ -0,0 +1,26 @@
+name: Auto Assign to Project(s)
+
+on:
+  pull_request_target:
+    types: [opened, labeled]
+env:
+  MY_GITHUB_TOKEN: ${{ secrets.AUTO_ASSIGN_WORKFLOW_TOKEN }}
+
+jobs:
+  assign-to-project:
+    runs-on: ubuntu-latest
+    name: Assign to Project
+    steps:
+    - name: Assign pull requests to "Unprioritised"
+      uses: srggrs/assign-one-project-github-action@1.2.0
+      if: github.event.action == 'opened'
+      with:
+        project: https://github.com/orgs/simple-icons/projects/2
+        column_name: Unprioritised
+
+    - name: Assign `icon outdated` pull requests to "Priority 1"
```

Minor nitpick, but I suggest we remove this extra newline to keep the formatting consistent with [existing workflow files](https://github.com/simple-icons/simple-icons/tree/880c706a50e4d3101dcb869d04d88ea2fcb0a77c/.github/workflows)

```suggestion
        column_name: Unprioritised
    - name: Assign `icon outdated` pull requests to "Priority 1"
```